### PR TITLE
build r-base against 3.4.1 and 3.5.1

### PR DESCRIPTION
**Pull request by bgruening**
blocked by https://github.com/conda-forge/r-base-feedstock/pull/56

Checklist
* [x] Used a fork of the feedstock to propose changes
* [ ] Bumped the build number (if the version is unchanged)
* [x] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/conda_smithy.html#how-to-re-render ) with the latest `conda-smithy`
* [x] Ensured the license file is being packaged.

### CODE REVIEW

**Comment by bgruening** [comment]
@conda-forge-admin, please rerender

File: recipe/conda_build_config.yaml [hunk 1]
```diff
@@ -414,6 +414,7 @@ readline:
   - 7.0
 r_base:
   - 3.4.1
+  - 3.5.1
```
Comment (ocefpaf) [inline recipe/conda_build_config.yaml]: @CJ-Wright this would be the "big migration" (besides the compilers+Python 2.7) test you are looking for :smile:

Comment (CJ-Wright) [reply recipe/conda_build_config.yaml]: lol, I was just hoping to look into the future so we can plan for the migrations. I wasn't expecting someone to build a time machine and make the future now!

Comment (CJ-Wright) [reply recipe/conda_build_config.yaml]: :relieved:

**Comment by isuruf** [comment]
Isn't this blocked by the r-base PR mentioned above?

**Comment by ocefpaf** [comment]
> Isn't this blocked by the r-base PR mentioned above?

I though we already had a version of 3.5.1 but I guess we need that version.

@bgruening are we close on that one? I cannot help at the moment b/c I'm in transit but I can take a look at it next week.

**Comment by bgruening** [comment]
@ocefpaf we do have r 3.5.1 but we agreed at some point to only rebuild all packages with the new compilers. But as it turns out this still takes some time. So depending on how long it might be better to start rebuilding? I'm not sure here. Currently, we have a pkg but are not using it.

**Comment by bgruening** [comment]
We should probably revert this IFF we want to wait for the rebuild.

**Comment by ocefpaf** [comment]
> We should probably revert this IFF we want to wait for the rebuild.

Let's assess if we can solve https://github.com/conda-forge/r-base-feedstock/pull/57 in the next few day first. If not we revert this one.

**Comment by bgruening** [comment]
The problem is that we need to start at least the new-compiler rebuilds :(

**Comment by CJ-Wright** [comment]
I'm hoping the compiler migration will start soon. I'm happy this got in since we need as large if a sub graph as possible to make sure things are rebuilt in the correct order.

**Pull request merged**